toxdns/toxdns.c

```diff
@@ -37,14 +37,14 @@ static const char base32[32] = {
 
 #define _encode(a, b, c) \
 { \
-uint8_t i = 0; \
-    while(i != c) { \
+    uint8_t _i = 0; \
```

Comment:
Avoid shadowing of i, which is often already used. Avoids confusion about which i is used where.